src/main/java/com/acme/dbo/txlog/ByteCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.acme.dbo.txlog;
+
+import static java.lang.Math.abs;
+
+public class ByteCommand {
+        private static String DECOR = "primitive: ";
+
+        public Byte currentValue;
+        private static String accumulator;
```

Comment:
static? Nope! we have to have _instance_ variables not class-wide variables. Your code works fine until we have to create more that one Controller instance. After it both of them actually will have one state.

Reply:
Done.